### PR TITLE
Updated VignetteBuilder field to include knitr and rmarkdown

### PR DESCRIPTION
**Pull request by harshagr70**
This PR updates the DESCRIPTION file to include both knitr and rmarkdown in the VignetteBuilder field. This change ensures that all necessary dependencies for building vignettes are explicitly declared.

Motivation and Context
This change addresses issue #3392 to prevent errors when building vignettes that depend on rmarkdown.

### CODE REVIEW

File: modules/data.atmosphere/DESCRIPTION [hunk 1]
```diff
@@ -80,6 +80,6 @@ License: BSD_3_clause + file LICENSE
 Copyright: Authors
 LazyLoad: yes
 LazyData: FALSE
-VignetteBuilder: knitr
+VignetteBuilder: knitr, rmarkdown
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION:84]: To fix the current  `check/modules` CI failure, add rmarkdown to the Suggests section, between current lines 71 and 72 above. (I thought all packages already had rmarkdown in Suggests, but apparently we missed this one).

Comment (harshagr70) [reply modules/data.atmosphere/DESCRIPTION:84]: Thanks for the feedback! I’ve added rmarkdown to the Suggests section as required .

**Review by infotroph — APPROVED** [review]
Thanks! This looks great.

I forgot to mention the last step, which was to run `./scripts/generate_dependencies.R` after editing the Suggests section.  I went ahead and did that and will merge as soon as CI finishes.

**Comment by infotroph** [comment]
The remaining CI failures are an unrelated known issue (Docker deployment only works right from same-fork branches atm) and do not need to delay merge.

**Pull request merged**